source/05-EL/01.ptx

```diff
@@ -953,6 +898,65 @@
       </task>
     </activity>
 
+        
+    <remark xml:id="remark-number-e">
+      <statement>
+        <p>
+          Similar to how <m>\pi</m> arises naturally in geometry, there is an irrational number called <term><m>e</m></term> that
+          arises naturally when working with exponentials.  We usually use the approximation  <m>e \approx 2.718282</m>.  
+          <m>e</m> is also found on most scientific and graphing calculators. 
+        </p> 
+      </statement> 
+      <aside>
+        <p> See <xref ref="activity-motivate-e"/> for an exploration of how <m>e</m> arises naturally.
```

Comment:
```suggestion
        <p> See <xref ref="act-evaluate-base-e"/> for an exploration of how <m>e</m> arises naturally.
```

perhaps?

Reply:
No, this was intentional, I guess this is blocked by #314

Reply:
Yeah, we got our wish a couple weeks ago that CI/CD will be more agressive in blocking errors, such as non-existent xml-ids.

I think the solution then is to put a stub activity with that ID into this PR.